bundles/org.openhab.binding.plugwiseha/src/main/resources/OH-INF/thing/channels.xml

```diff
@@ -185,14 +213,22 @@
 		<label>Domestic Hot Water Setpoint Temperature</label>
 		<description>Gets the temperature of the domestic hot water setpoint</description>
 		<category>heating</category>
+		<tags>
+			<tag>Setpoint</tag>
+			<tag>Temperature</tag>
+		</tags>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
 
 	<channel-type id="maxBoilerTemperature">
 		<item-type>Number:Temperature</item-type>
 		<label>Max Boiler Temperature</label>
-		<description>Gets the maximum temperature ofthis boiler</description>
+		<description>Gets the maximum temperature of this boiler</description>
```

Comment:
👍 Please also update in I18N properties file.

Reply:
Thanks